references/recognition/train_pytorch.py

```diff
@@ -235,6 +259,7 @@ def main(args):
                 T.RandomApply(T.ColorInversion(), 0.9),
             ]),
         )
+    val_hash = None
```

Comment:
remove this please - because otherwise we would override the val_hash from --val_path passed datasets